docs/man/docker.1.md

```diff
@@ -292,6 +295,18 @@ for data and metadata:
       --storage-opt dm.metadatadev=/dev/vdc \
       --storage-opt dm.basesize=20G
 
+# EXEC DRIVER OPTIONS
+
+Options to exec backend can be specified with **--exec-opt** flags. The
+only backend which currently takes options is *native* (libcontainer). Therefore
+use these flags with **-s=**native.
+
+Here is the list of *native* options:
```

Comment:
lol sorry grammar
I probably would  not have gotten into college if it wasnt for my math sat score. no joke, my grammar suck, i will update